assets/src/styles/pages/_monitoring-location.scss

```diff
@@ -18,7 +18,7 @@
   @include u-padding-bottom(1);
 }
 
-#dv-accordion {
+.usa-accordion {
```

Comment:
Changing this will cause all accordions to have that margin and that may not be desirable.

Reply:
Is there a reason we would like the accordions to be different? There were several different margins applied, some using 'em' and some using USWDS units, making them slightly uneven in spacing. This standardizes them and can always be overridden if the odd case arises.

Reply:
This particular one has been put is specifically to style the dv-accordion. (Groundwater Data). I would be fine with removing it entirely if it does not affect the display of the dvHydrograph.